Bolt.php

```diff
@@ -134,7 +134,7 @@ private function handshake(): bool
 
         $this->unpackProtocolVersion();
         if (empty($this->version)) {
-            $this->error('Wrong version');
+            static::error('Wrong version');
```

Comment:
Everywhere is error function called with Bolt prefix, it will be good to keep it that way

```suggestion
            Bolt::error('Wrong version');
```